Mimir.Worker/BlockPoller.cs

```diff
@@ -17,10 +17,10 @@ public async Task RunAsync(CancellationToken cancellationToken)
         var stateGetter = new StateGetter(stateService);
         while (!cancellationToken.IsCancellationRequested)
         {
-            var syncedBlockIndex = await mongoDbStore.GetLatestBlockIndex();
             var currentBlockIndex = await stateService.GetLatestIndex();
+            var syncedBlockIndex = await mongoDbStore.GetLatestBlockIndex() ?? currentBlockIndex - 1;
```

Comment:
The LatestBlockIndex in the code path is set during the scraping process. I can't guarantee the validity of the data with or without this value when transactions are not available, as they are now, but I feel that not even having the LatestBlockIndex means that we have to start over from the Scrapper step. In the case of BlockPoller, it's very dangerous to arbitrarily skip block indices because it follows block by block and synchronizes the state.

<!-- 코드 경로 상 LatestBlockIndex는 스크래핑을 과정에서 설정됩니다. 현재 처럼 트랜잭션을 사용할 수 없는 상황에서는 이 값이 있던 없던 데이터의 유효성을 보장할 수 없지만 LatestBlockIndex 조차 없는 것은 Scrapper 단계 부터 다시 시작해야함을 의미한다고 느껴집니다. BlockPoller의 경우 블록 단위로 따라가면서 상태들을 동기화 하기 때문에 임의로 블록 인덱스를 스킵하는 것은 굉장히 위험한 일입니다. -->

Reply:
It seems a bit awkward to me to have to consider the scraping step at the `BlockPoller` level.
However, I agree that the responsibility of covering all blocks at the `Mimir.Worker` level is more important.